index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Martin Karugaba</title>
+    <link rel="stylesheet" href="../dist/output.css" />
+  </head>
+  <body class="font-poppins">
+    <!--* navbar -->
+    <nav
+      class="h-14 w-full px-5 py-4 bg-slate-50 flex justify-between items-center"
+    >
+      <!--* logo -->
+      <div
+        class="text-light-blue leading-3 gap-y-0 flex flex-col justify-between items-start cursor-pointer font-semibold"
+      >
+        <p class="text-2xl">Martin</p>
+        <p class="-mt-[10px] text-2xl">Karugaba</p>
+      </div>
+
+      <!--* nav links -->
+      <div
+        class="w-[26px] h-[30px] border-white flex flex-col justify-center items-center cursor-pointer"
+      >
+        <!--* hamburger -->
+        <div class="h-[3px] w-[100%] bg-light-blue rounded-sm"></div>
+        <div
+          class="h-[3px] w-[100%] bg-light-blue my-[6px] rounded-sm"
+        ></div>
+        <div class="h-[3px] w-[100%] bg-light-blue rounded-sm"></div>
+      </div>
+    </nav>
+    <!--* end navbar -->
+
+    <!--* header -->
+    <section
+      class="h-[90vh] bg-hero-shapes bg-cover flex flex-col justify-center gap-y-4 items-start px-5"
+    >
+      <h1 class="text-dark-blue font-bold text-[40px] leading-11">
+        I'm Martin Karugaba <br />
+        Glad to see you!
+      </h1>
+      <p class="text-base leading-7 text-light-teal">
+        I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and
+        experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+
+      <div>
+        <h2 class="uppercase text-4xl text-h2 font-medium">Let's connect</h2>
+
+        <!--* icons -->
+        <div class="flex justify-between items-center gap-y-4 mt-2">
+          <!--* twitter icon -->
+          <a href="https://twitter.com/martin_karugaba">
+            <img
+              class="h-[20px] w-[20px]"
+              src="./img/twitter.png"
+              alt="twitter icon"
+            />
+          </a>
+
+          <!--* linkedin icon -->
+          <a
+            href="https://www.linkedin.com/in/martin-karugaba-822442173/"
+          >
+            <img
+              class="h-[20px] w-[20px]"
+              src="./img/linkedin.png"
+              alt="linkedin icon"
+            />
+          </a>
+
+          <!--* github icon -->
+          <a href="https://github.com/martinkarugaba">
+            <img
+              class="h-[20px] w-[20px]"
+              src="./img/github.png"
+              alt="github icon"
+            />
```

Comment:
- [ ] I think it's better to use an order list `ul and li` instead of using a `div.` For user experience(UX) and semantic reasons for all the social media icons.